neo/rawio/intanrawio.py

```diff
@@ -188,7 +191,7 @@ def _parse_header(self):
                 dtype_size = np.dtype(stream_datatype).itemsize
                 n_samples = size_in_bytes // (dtype_size * num_channels)
                 signal_stream_memmap = np.memmap(
-                    file_path, dtype=stream_datatype, mode="r", shape=(num_channels, n_samples)
+                    file_path, dtype=stream_datatype, mode="r", shape=(num_channels, n_samples), order='F',
```

Comment:
why not 
```suggestion
                    file_path, dtype=stream_datatype, mode="r", shape=(n_samples, num_channels), order='C',
```

Reply:
Good point. Let me simplify. We originally were reimplementing the Matlab function from the Intan docs, so there is also a `.T` of the map that we need to remove.